assets/chezmoi.io/docs/user-guide/frequently-asked-questions/encryption.md

```diff
@@ -100,3 +100,54 @@ $ chezmoi add --encrypt ~/.ssh/id_rsa
 When you run `chezmoi init` on a new machine you will be prompted to enter your
 passphrase once to decrypt `key.txt.age`. Your decrypted private key will be
 stored in `~/.config/chezmoi/key.txt`.
+
+## How to migrate between encryption tools?
+
+To migrate from gpg to age encryption or vise versa, one can follow these steps:
```

Comment:
> migrate from gpg to gpg

Nice, I haven't thought about that case.

Reply:
It's an occupational hazard of dealing with operational security. It would be *nice* if chezmoi offered a way to rekey without the files touching disk, but given that they always hit disk anyway, this is a reasonable workflow.